packages/core/upload/admin/src/utils/getFolderURL.js

```diff
@@ -1,11 +1,11 @@
 import { stringify } from 'qs';
 
-const getFolderURL = (pathname, query, folder) => {
+const getFolderURL = (pathname, query, folderID) => {
```

Comment:
worth adding a JSdoc & clarifying that `folderID` can be `undefined` and thats okay?